Code/Legacy/CryCommon/platform.h

```diff
@@ -135,15 +135,6 @@
 
 #include <AzCore/PlatformDef.h>
 
-#if defined(AZ_MONOLITHIC_BUILD)
-    #define DLL_EXPORT
-    #define DLL_IMPORT
```

Comment:
The `PlatformIncl.h` include on line 34 of this file already includes `PlatformDef.h`.

I think the PlatformDef.h include here was just to get access to the AZ_DLL_EXPORT/AZ_DLL_IMPORT macros, which is no longer needed here

Reply:
turns out platformdef.h is included multiple times into this file
I'll move it to the top though and see what happens.